index.html

```diff
@@ -208,5 +208,30 @@ <h4>Skills</h4>
                 </ul>
                 <a href="#"><h5>tyaps002@gmail.com</h5></a>
             </section>
+            
+            <section class="form-survey">
+                <form action="https://formspree.io/f/mpzbbknz"  method="POST" class="form">
+                    <p>I'm always interested in hearing about new projects, 
```

Comment:
- [ ] please use the same font weight, family and size for the text so it looks more like the figma version

Reply:
Done